system/kube-monitoring/charts/prometheus-collector/templates/_prometheus.yaml.tpl

```diff
@@ -256,6 +256,7 @@ scrape_configs:
 {{ range $region := .Values.global.regions }}
 - job_name: 'blackbox-ingress-{{ $region }}'
   metrics_path: /probe
+  region_probed_from: {{ $region }}
```

Comment:
won't work. unlike `metrics_path` this is not a keyword given by prometheus. we want to add it in the relabel section